src/nomad_simulations/schema_packages/base_sections.py

```diff
@@ -0,0 +1,49 @@
+from nomad.datamodel.data import ArchiveSection
+from nomad.metainfo import Quantity, Reference
+
+
+class ModelBaseSection(ArchiveSection):
```

Comment:
just a small note: the terminology "model" here is a bit confusing to me considering the other uses of model within our schema